clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1623,6 +1629,8 @@ protected void processCommonConsumerAction(
         partitionConsumptionStateMap.remove(partition);
         storageUtilizationManager.removePartition(partition);
         kafkaDataIntegrityValidator.clearPartition(partition);
+        // Reset the error partition tracking
+        partitionIngestionExceptionList.set(partition, null);
```

Comment:
should we check the content and print some log that we are unsubbing an ERROR parittion ?

Reply:
There must be error logs before un-subscription if there is an exception happened.

Reply:
Like subscribe action prints `  LOGGER.info(              "{} subscribed to: {} Offset {}",` We can modify the message at the beginning and move to the end and add some indication at we unsubbed an error partition. I believe it would be helpful.

Reply:
Sure.